packages/frontend/@n8n/design-system/src/v2/components/DropdownMenu/DropdownMenuItem.vue

```diff
@@ -0,0 +1,375 @@
+						ref="searchRef"
+						:model-value="searchTerm"
+						:placeholder="searchPlaceholder ?? 'Search...'"
+						:show-icon="showSearchIcon !== false"
+						@update:model-value="handleSearchUpdate"
+						@key:escape="closeSubMenu"
+						@key:navigate="subMenuNavigation.navigate"
+						@key:arrow-left="subMenuNavigation.handleArrowLeft"
+						@key:enter="subMenuNavigation.handleEnter"
+					/>
+
+					<div v-if="loading" :class="$style['loading-container']">
+						<N8nLoading
+							v-for="i in loadingItemCount"
+							:key="i"
+							:rows="1"
+							:class="$style['loading-item']"
+							variant="p"
+						/>
+					</div>
+					<template v-else-if="hasChildren">
+						<div :class="$style['children-container']">
+							<template v-for="(child, childIndex) in props.children" :key="child.id">
+								<N8nDropdownMenuItem
+									v-bind="child"
+									:highlighted="searchable && subMenuHighlightedIndex === childIndex"
+									@select="handleSelect"
+									@search="handleChildSearch"
+								/>
+							</template>
+						</div>
+					</template>
+
+					<!-- Empty state when search returns no results -->
+					<div v-else-if="searchable && searchTerm" :class="$style['empty-state']">No results</div>
+				</DropdownMenuSubContent>
+			</DropdownMenuPortal>
+		</DropdownMenuSub>
+
+		<!-- Regular item without children -->
+		<DropdownMenuItem
+			v-else
+			:disabled="disabled"
+			:class="[$style.item, props.class, { [$style.highlighted]: highlighted }]"
+			@select="handleItemSelect"
+		>
+			<slot name="item-leading" :item="props" :ui="leadingProps">
+				<Icon
+					v-if="icon?.type === 'icon'"
+					:icon="icon.value"
+					:class="[$style['item-leading'], $style.icon]"
+					size="large"
+					:color="disabled ? 'text-xlight' : 'text-light'"
+				/>
+				<span v-else-if="icon?.type === 'emoji'" :class="[$style['item-leading'], $style.emoji]">
+					{{ icon.value }}
+				</span>
+			</slot>
+			<N8nText
+				:class="$style['item-label']"
+				size="medium"
+				:color="disabled ? 'text-light' : 'text-dark'"
+			>
+				{{ label }}
+			</N8nText>
+			<slot name="item-trailing" :item="props" :ui="trailingProps" />
+			<Icon
+				v-if="checked"
+				icon="check"
+				:class="$style['item-check']"
+				size="large"
+				:color="disabled ? 'text-xlight' : 'text-light'"
+			/>
+		</DropdownMenuItem>
+	</div>
+</template>
+
+<style module>
+.wrapper {
+	display: contents;
+}
+
+.children-container {
+	padding: var(--spacing--4xs);
+}
+
+.item {
+	font-size: var(--font-size--2xs);
+	line-height: 1;
+	border-radius: var(--radius);
+	display: flex;
+	align-items: center;
+	min-height: var(--spacing--lg);
+	padding: var(--spacing--2xs);
+	position: relative;
+	user-select: none;
+	color: var(--color--text--shade-1);
+	gap: var(--spacing--3xs);
+	outline: none;
+
+	&:not([data-disabled]) {
+		&:hover,
+		&[data-highlighted],
+		&.highlighted {
+			background-color: var(--color--background--light-1);
```

Comment:
Use `var(--color--foreground--tint-1)` the current color isn't strong enough in light mode